src/components/ProfileUI/ProfileUI.jsx

```diff
@@ -1,16 +1,20 @@
 import React from "react";
 
-const ProfileUI = ({ name, surname }) => (
+const ProfileUI = ({ name, surname, profile_pic }) => (
```

Comment:
can be profile picure

Reply:
I kept it the same as we named it in the backend, for better understanding.

Reply:
This is a lot better than  I would have done it @marcopeg. It's fixed now